themes/default/content/docs/intro/concepts/config.md

```diff
@@ -240,3 +240,19 @@ Console.WriteLine($"Active: {data.GetProperty("active")}");
 {{% /choosable %}}
 
 {{< /chooser >}}
+
+## Special Configuration Options
+
+This is a list of configuration keys that the Pulumi CLI is aware of:
+
+### `pulumi:disable-default-providers`
+
+A list of packages for which default providers should be disabled. `*` disables default providers for all packages.
```

Comment:
If you take my suggestion to put the new content in the providers page in a section, might be nice to link this text.


```suggestion
A list of packages for which [default providers should be disabled]({{< relref "/docs/intro/concepts/resources/providers#disabling-default-providers" >}}). `*` disables default providers for all packages.
```